scripts/make_windows.py

```diff
@@ -1,40 +1,68 @@
+#!/usr/bin/env python3
+#
+# make_windows.py
+# Inputs: a stream of input ("-", stdin) from samtools and a threshold (integer)
+# Sums stream of input depth until cumulative coverage exceeds threshold, and makes a new bin when that occurs 
+# Outputs bins to stdout
+#
```

Comment:
FYI - for python it's better if this is in a triple quote string at the top of the file.  More standard and get's added as a `__doc__` property
In python triple quotes are multiline strings.

```
#!/usr/bin/env python3
"""
make_windows.py
Inputs: a stream of input ("-", stdin) from samtools and a threshold (integer)
Sums stream of input depth until cumulative coverage exceeds threshold, and makes a new bin when that occurs 
Outputs bins to stdout
"""
```